include/cynq/xrt/memory.hpp

```diff
@@ -0,0 +1,67 @@
+/*
+ * See LICENSE for more information about licensing
+ *
+ * Copyright 2023
+ * Author: Luis G. Leon-Vega <luis.leon@ieee.org>
+ *         Diego Arturo Avila Torres <diego.avila@uned.cr>
+ *
+ */
+#pragma once
+#include <memory>
+
+#include <cynq/enums.hpp>
+#include <cynq/memory.hpp>
+#include <cynq/status.hpp>
+
+namespace cynq {
+class XRTMemory : public IMemory {
```

Comment:
Undocumented class.

Provide details that it is based on the Buffer Object from XRT